server/src/dev.rs

```diff
@@ -0,0 +1,13 @@
+mod server;
```

Comment:
I feel that we can eliminate redundancy of code across environments. Any reason why prod.rs would not work for local development?

Reply:
Because we want to use the webpack dev server for the HTTP part, but rust for the websocket part in development

The webpack dev server gives us a fast edit/refresh cycle when modifying the client/ and www/ directories